plugins/comments.md

```diff
@@ -161,3 +184,14 @@ Follow this procedure to delete all conversations in the document:
 **Result**: All the comments for the selected document will be deleted.
 
 Check out the [Comments demo]({{site.baseurl}}/demo/comments-2) to try this new feature.
+
+## Using Comments 2.0 embedded mode with the Full Page plugin
+
+Users have to be cautious when deciding the order in which the plugins are added in the plugins list.
+
+Comments can cause an issue if the Full Page plugin `fullpage` appears before Comments 2.0 plugin `tinycomments` in the plugin list, and "tinycomments" is configured to use `embedded mode`.
+
+The consequence of this situation is that when a saved document is re-opened, the comment data is lost (but the highlights are still there). This is because the comment data is in the wrong place. The order that the plugins appear affects the order that the `getContent` hooks are processed in. The `fullpage` adds outer elements to the content and then `tinycomments` adds its comment data outside those markers. `tinycomments` expects to find its data inside the content, but having `fullpage` add outer `<html>` elements before `tinycomments` adds its comment data. This results in the comment data being lost
```

Comment:
adds outer*

Reply:
The last sentence is a bit confusing. Maybe change to 

```
The `fullpage` adds outer elements to the content and then `tinycomments` adds its comment data outside those markers, which puts them in the wrong place
```

Reply:
Fixed